core/pdfbulkexport.class.inc.php

```diff
@@ -275,7 +280,7 @@ public function GetFileExtension()
 	 * @return string Rendered value of $oAttDef / $oValue according to the desired $sOutputType
 	 * @since 2.7.8
 	 */
-	protected function GetAttributeImageValue(AttributeImage $oAttDef, ormDocument $oValue, string $sOutputType)
+	protected function GetAttributeImageValue(DBObject $oObj, AttributeImage $oAttDef, ormDocument $oValue, string $sOutputType)
```

Comment:
```suggestion
	protected function GetAttributeImageValue(DBObject $oObj, string $sAttCode, string $sOutputType)
```

If we have to pass the whole object, then passing the att. def. and the value makes no sense anymore. We should only pass the object and the att. code IMO.

_Mind to update the PHPDoc and the consumers as well_

Reply:
Updated the signature with 06259546
Indeed getting again attdef and value shouldn't be that bad as everything is in memory.
Consumers were already updated on last modification, I did it also for this modification.